feat: filter metadata by brand; hardcode to ainslie for now

rahdoht opened the pull request:
Not feature complete, but I wanted to demonstrate how you might filter the metadata by a given trait_type, such as Brand === Ainslie, to allow the user to be fully inundated with the analysis paralysis of modern consumerism. Adding drop down menus or autocomplete text fields to provide the user with the unbearable lightness of choice is not my forte, but hopefully you will be able to incorporate that easily.

Thanks for reviving cigbot!

karpapy commented:
ay lovely, will add this weekend. the people asketh and we shall giveth. great work on the original cigbot ❤️

karpapy commented:
Picking up the convo from twitter, @rahdoht I think u should delete this fork so that all the future metadata isn't out in the open 😅 and open a new PR with just the minted metadata for now

rahdoht commented:
I have pinched myself into a pickle. I jumped the gun and deleted my repo, thinking that it might wipe this off the webs. When that didn't work, I started digging into SO posts. It's slightly difficult to **delete** a PR on github; we'll have to ask them to do it and claim that secret credentials are at stake, which is almost true.

The alternative is to try to re-open this PR and then wipe the history and force push to overwrite the commits. The PR would stay but it would be empty. Except that I'm not sure I can re-open the PR now that I've deleted my original fork of your repo. But if you see any way to re-open the PR, please do so and then I can try my hand at the git wizardry.
